src/zenml/integrations/kubeflow/container_entrypoint.py

```diff
@@ -475,7 +476,19 @@ def main() -> None:
     # available
     integration_registry.activate_integrations()
 
-    metadata_store = Repository().active_stack.metadata_store
+    try:
+        repo = Repository()
+    except RepositoryNotFoundError:
+        raise RepositoryNotFoundError(
+            "Unable to find ZenML repository in docker container. Make sure "
+            "your ZenML repository is not included in the dockerignore file "
+            "used when building the docker image to run this pipeline using "
+            "Kubeflow Pipelines. This dockerignore might be specified using "
+            "the `@pipeline` decorator or a file named '.dockerignore' at the "
+            "root of your local ZenML repository."
```

Comment:
Not every user might know what the ZenML repository is - might be worth specifically saying .zen directory at some point in the message.


```suggestion
            "Unable to find ZenML repository in docker container. Make sure "
            "your ZenML repository (.zen directory) is not included in the dockerignore file "
            "used when building the docker image to run this pipeline using "
            "Kubeflow Pipelines. This dockerignore might be specified using "
            "the `@pipeline` decorator or a file named '.dockerignore' at the "
            "root of your local ZenML repository."
```

Reply:
Good idea, I'll fix it without the github suggestion to make sure the text doesn't overlap our 80 character line limit :)